packages/common/src/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -43,14 +43,31 @@ export interface RangeMark {
   excludeActive?: boolean;
 }
 
+interface PartialPosition {
+  readonly line: number;
+  readonly character: number;
+}
+
+interface PartialRange {
+  readonly start: PartialPosition;
+  readonly end: PartialPosition;
+}
+
+export interface ExplicitMark {
+  type: "explicit";
```

Comment:
I think I'd be a bit more explicit (😊) and call this `explicitRange`.  I'd also change

- `ExplicitMark` to `ExplicitRangeMark` and
- `ExplicitMarkStage` to `ExplicitRangeMarkStage`

WDYT?

Reply:
agreed